src/screens/login/loginContainer.js

```diff
@@ -55,7 +55,8 @@ class LoginContainer extends Component {
 	 * after the component mounted (to clean the state). also triggers the auto-login if 
 	 * configured in appConfig.js
 	 */
-	componentDidMount = () => {
+	componentDidMount = async () => {
```

Comment:
why did you change it to 'async'?

Reply:
at one point I had problems getting the timing right for the automated login (should it be activated and running in development mode). so i tried to run the event handler asynchronously, which actually worked out - but the solution provided by React Native Testing Library (using the `waitFor()` function) is in my eyes much better as the code we want to test does not need to be changed in any way for that to work out.

so short answer: missed to undo it